contrib/babelfishpg_tsql/antlr/TSqlLexer.g4

```diff
@@ -32,6 +32,7 @@ lexer grammar TSqlLexer;
 @header
 {
     extern bool pltsql_quoted_identifier;
+    extern bool pltsql_parseonly;
```

Comment:
Why do we need this in TSqlLexer file?

Reply:
Needed to include this here so that the variable is visible from the parser code.